.github/actions/phar/action.yaml

```diff
@@ -18,15 +18,15 @@ runs:
       shell: bash
       working-directory: tools/phar
 
-    - name: Compile phar Linux
+    - name: Build Castor PHAR Archive for Linux
```

Comment:
I used to write `PHAR` but I was wrong. It's `phar`.
And phar archive is a bit redundant, since phar means `PHP Archive`

https://www.php.net/manual/en/intro.phar.php